apps/rush-lib/assets/rush-init/common/config/rush/version-policies.json

```diff
@@ -77,7 +77,13 @@
      * inappropriately.  The minor/patch version parts will be bumped independently according
      * to the types of changes made to each project, according to the "rush change" command.
      */
-    "lockedMajor": 3
+    "lockedMajor": 3,
+
+    /**
+     * (Optional) If false, developers of the specified version policy will no longer
+     * be prompted for changelog files during the rush change command.
+     */
+    "noChangeLogRequired": false
```

Comment:
Let's rename this to `exemptFromRushChange`.  It more clearly explains the effect, and avoids a double-negative `noIceCream = false` (*What!? Do I get 🍦 or not!!*)